### ISSUE TITLE
The manifest got some git leftovers?

### ISSUE THREAD

**Issue by budimanjojo** [OT-CONTAINER-KIT/redis-operator#251]
https://github.com/OT-CONTAINER-KIT/redis-operator/blob/aeecf234b56e4ccd7ca5a41d42c31b5366cc80c8/config/crd/bases/redis.redis.opstreelabs.in_redisclusters.yaml#L3428
https://github.com/OT-CONTAINER-KIT/redis-operator/blob/aeecf234b56e4ccd7ca5a41d42c31b5366cc80c8/config/crd/bases/redis.redis.opstreelabs.in_redisclusters.yaml#L3654
The file `config/crd/bases/redis.redis.opstreelabs.in_redisclusters.yaml` of `v0.10.0` is broken. Looks like the file is 3 times larger than master and there are a lot lines I pointed out above. Applying with kustomize will cause this error:
```
 error: accumulating resources: accumulation err='accumulating resources from 'https://raw.githubusercontent.com/OT-CONTAINER-KIT/redis-operator/v0.10.0/config/crd/bases/redis.redis.opstreelabs.in_redisclusters.yaml': yaml: line 3426: could not find expected ':'': git cmd = '/usr/bin/git fetch --depth=1 origin HEAD': exit status 128
```

**Comment by iamabhishek-dubey** [comment]
Fixed in https://github.com/OT-CONTAINER-KIT/helm-charts/commit/e048aebc17aaa79e9244135ad7315ce299d49ea8

**Comment by budimanjojo** [comment]
Sorry but I think you don't understand my real issue. The problematic manifest is in `v0.10.0` tag and not `master` branch. Can you release a new version or backport the fix to `v0.10.0`? Because I'm using versioned gitops tool to deploy this.

**Comment by szamuboy** [comment]
I think this issue has been prematurely closed. `v0.10.0` is not usable.
